sdk/data/azcosmos/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+* Fixed ReplaceThroughput operations on Database and Container
```

Comment:
In other SDKs, we also include the PR link in the changelog for reference for customers, like this - https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-cosmos/CHANGELOG.md

Reply:
In Go, Changelogs do not have PR links. For example, the Azure Core: https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azcore/CHANGELOG.md

@jhendrixMSFT what are your thoughts on including PR links in the changelog?

Reply:
I know some Go SDKs include the PR, but we haven't formalized any policy (maybe we should). It is a slight inconvenience as you have to update the PR comment once you have it, thus easy to miss (I have witnessed this multiple times).

I don't have strong feelings on it at present, and nobody has complained that this information is missing (I can find the info in git history just fine). That said, please feel free to include this info if you (or customers) would find it useful.

Reply:
@ealsur to that point, even other SDKs in Azure core don't have, this is something specific to Cosmos. I am fine either way as well, thought we used to have it in .Net SDK as well, and we made it consistent across Java, python and JS.

Reply:
I'll add it then. For Go my intent was first adhere with the Go team directives, since there is no pushback, I will add it.

Reply:
```suggestion
* Fixed ReplaceThroughput operations on Database and Container. See [PR 22923](https://github.com/Azure/azure-sdk-for-go/pull/22923)
```